pxr/usd/usdGeom/schema.usda

```diff
@@ -2292,6 +2292,36 @@ class Camera "Camera" (
     However, it follows that if even one property is authored in the correct
     scene units, then they all must be.
 
+    \\section UsdGeom_CameraExposure Camera Exposure Model
+
+    UsdGeomCamera models exposure by a camera in terms of exposure time, ISO,
+    f-number, and exposure compensation, mirroring the controls on a real camera.
+    These parameters are provided by \\ref UsdGeomCamera::GetExposureTimeAttr(),
+    \\ref UsdGeomCamera::GetExposureIsoAttr(), \\ref UsdGeomCamera::GetExposureFNumberAttr(), 
+    and \\ref UsdGeomCamera::GetExposureAttr(), respectively. 
+    \\ref UsdGeomCamera::GetExposureResponsivityAttr() provides an additional scaling
+    factor to model the overall responsivity of the system, including response of 
+    the sensor and loss by the lens.
+
+    The calculated scaling factor can be obtained from 
+    \\ref UsdGeomCamera::GetExposureScale(), which is also the value provided by 
+    \\ref HdCamera::GetExposure(). It is computed as:
+    \\code
+        exposureScale = exposureResponsivity * 
+            (exposureTime * (exposureIso/100) * pow(2, exposure)) 
+            / (exposureFStop * exposureFStop)
+    \\endcode
+
+    This scaling factor is combined from two parts: The first, known as the __imaging ratio__ 
+    (in _steridian-second_), converts from incident luminance at the front 
+    of the lens system, in _nit_ (_cd/m^2_), to photometric exposure at the sensor
+    in _lux-second_. The second, <tt>exposureResponsivity</tt> 
```

Comment:
since you're already using markdown for formatting, should be able to use backticks rather than `<tt>` html?